docs/rpc-api.md

```diff
@@ -62,18 +62,20 @@ Below is a table of provided methods.
 | [eth_chainId](https://besu.hyperledger.org/en/stable/Reference/API-Methods/#eth_chainid)                                                  | [Chain_ID](../README.md#configuration)    | Relay Only    |
 | [eth_coinbase](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_coinbase)                                                       | `-32601`                                  | N/A   |
 | [eth_estimateGas](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_estimategas)                                                 | N/A                                       | Both Nodes  |
-| [eth_feeHistory](https://besu.hyperledger.org/en/stable/Reference/API-Methods/#eth_feehistory)                                            | N/A                                       | Both Nodes  |
-| [eth_gasPrice](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gasprice)                                                       | N/A                                       | Both Nodes  |
+| [eth_feeHistory](https://besu.hyperledger.org/en/stable/Reference/API-Methods/#eth_feehistory)                                            | N/A                                       | Mirror Node, Consensus Node backup  |
+| [eth_gasPrice](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gasprice)                                                       | N/A                                       | Mirror Node, Consensus Node backup  |
 | [eth_getBalance](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getbalance)                                                   | N/A                                       | Consensus Node    |
```

Comment:
eth_getBalance queries only the mirror node

Reply:
Updated